strawberry/permission.py

```diff
@@ -132,13 +237,16 @@ def apply(self, field: StrawberryField) -> None:
             elif isinstance(field.type, StrawberryList):
                 self.return_empty_list = True
             else:
-                errror = PermissionFailSilentlyRequiresOptionalError(field)
-                raise errror
+                error = PermissionFailSilentlyRequiresOptionalError(field)
+                raise error
 
-    def _on_unauthorized(self, permission: BasePermission) -> Any:
+    def _on_unauthorized(self, permission: BasePermission, **kwargs: dict) -> Any:
```

Comment:
This is probably a mistake - with kwargs you annotate value type (e.g. `int` instead of `dict[str, int]`, not sure what was intended here 🙂

Reply:
If there are no particular operations performed on kwargs values you could annotate them as `object`, as it's the most narrow type, `Any` being worse for that use case